addons/udes_advanced_packaging/views/stock_picking_type.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_picking_type_form_inherit" model="ir.ui.view">
+        <field name="name">stock.picking.type.form.inherit</field>
```

Comment:
Use `udes_advanced_packaging` at the end of the name instead of `inherit`. That way if we ever need to debug views it is a bit more clear which module the override came from.

Have asked Jack to do similar in another story for the `udes_stock_cron` module.